.github/workflows/examples-providers.yml

```diff
@@ -134,6 +134,8 @@ jobs:
       - name: test provider load
         if: ${{ matrix.project.test_deploy && matrix.wash-version == 'current' }}
         shell: bash
+        env:
+          RUST_LOG: trace
```

Comment:
Is this intended change to ship, or just helping diagnose CI while working through this?

Reply:
Hey Jonas, I am starting a thread in the #help channel, since I faced a few local reproduction blockers, definitely should not be here.